crates/starknet-devnet/tests/test_abort_blocks.rs

```diff
@@ -5,6 +5,7 @@ mod abort_blocks_tests {
     use server::api::json_rpc::error::ApiError;
```

Comment:
Once it is refactored to blockId, block tag tests will need to be added (latest, pending)